pkg/knuu/instance.go

```diff
@@ -808,6 +881,10 @@ func (i *Instance) StartWithoutWait() error {
 				return fmt.Errorf("error adding OpenTelemetry collector sidecar for instance '%s': %w", i.k8sName, err)
 			}
 		}
+		// gholi
+		if err := i.addNetworkConfigSidecar(); err != nil {
+			return fmt.Errorf("error adding network sidecar for instance '%s': %w", i.k8sName, err)
+		}
```

Comment:
This code adds sidecar to every instance, I removed the condition for this stage.
We need to decide on some config param that allows users to activate it. something like
```go
i := NewInstance(...).WithTrafficShaper()
```